content/docs/reference/validation_errors@es.md

```diff
@@ -768,7 +768,7 @@ Por ejemplo, `<amp-img src="" layout="responsive" width="42px" height="42rem">`
 
 ## Errores en las plantillas
 
-Las páginas de AMP no pueden incluir sintaxis de plantillas a menos que dicha sintaxis esté en una etiqueta de AMP específicamente diseñada para incluir plantillas, por ejemplo, [amp-mustache](/docs/reference/extended/amp-mustache.html).
+Las páginas de AMP no pueden incluir sintaxis de plantillas a menos que dicha sintaxis esté en una etiqueta de AMP específicamente diseñada para incluir plantillas, por ejemplo, [amp-mustache](/es/docs/reference/extended/amp-mustache.html).
```

Comment:
This link will break (no redirect).  Leave as is or fix to `es/docs/reference/components/amp-mustache.html`